docs/source/algorithms/GroupWorkspaces-v1.rst

```diff
@@ -45,7 +45,7 @@ Output
 
 .. testoutput::
 
-  Workspace's type is <class 'mantid.api._api.WorkspaceGroup'>
+  Workspace's type is <class '_api.WorkspaceGroup'>
```

Comment:
This doesn't feel right. Could you help me understand why this is not a fully qualified path now?

Reply:
I am looking into it, but I think it might be because before the structure always was `mantid.api._api`, in folder `bin/mantid/api/_api.pyd`. Right now the `_api.pyd` is just in `bin/`. 

This might also mean that the class name will be different between a packaged build and a developer build, but I haven't tested it